index.html

```diff
@@ -8,6 +8,44 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+    <div class="popular-destinations">
+        <div id="title-wrapper">
+            <div id="section-title">Our Popular Destinations</div>
+        </div>
+        <div class = "maps-and-destinations">
+            <div class="map">
+                <img src="./images/worldMap.png"/>
+            </div>
+            <div class="three-countries">   
+                    <div class="country">
+                        <img src = "./images/thailand.png" width="223px" height="223px"/>           
```

Comment:
remove the duplication for height and width for the images here and at lines 30 and 39.